src/pytest_regressions/common.py

```diff
@@ -188,3 +189,24 @@ def make_location_message(banner: str, filename: Path, aux_files: List[str]) ->
             else:
                 dump_aux_fn(Path(obtained_filename))
                 raise
+
+
+def round_digits(data: Union[list, dict], prescision: int) -> Union[list, dict]:
+    """
+    Recusrsively Round the values of any float value in a collection to the given prescision.
```

Comment:
```suggestion
    Recursively Round the values of any float value in a collection to the given precision.
```

Reply:
Minor: "Round" -> "round", "to the given precision" -> "to the given number of decimal places".